testsuite/tests/glbc/test_ingress_reconciliation.py

```diff
@@ -29,18 +27,15 @@ def backend_ingress(request, backend, blame):
 @backoff.on_exception(backoff.fibo, exception=httpx.ConnectError, max_time=600)
 def test_ingress_host_add(backend_ingress):
     """Creates ingress for a backend and checks for host field filled by glbc, checks host points to backend"""
-    rules = backend_ingress.rules
-    assert len(rules) == 1
-
-    backend_ingress.wait_for_hosts()
-    host = rules[0].get("host")
+    backend_ingress.wait_for_certificate()
```

Comment:
I'd go for a solution where we first wait for host and then for certificate as it would be more robust. (certificate generation can be disabled). and also split this test into two separate ones as this one should test that dns record has been set up properly and the other one should test that the certificate is generated and used properly. these are two test cases IMO. (yes I know the certificate would not work without generated hostname but I would still vote for separate solution)

Reply:
Host is there much sooner than certificate, almost instantly. To circumvent direct sleep I used certificate wait until the negative caching issue is fixed

Reply:
I don't like "almost instantly" since it can become flaky as this test is supposed to be the smoke one. and it is not clear that this wait should serve (even temporarily) as a replacement for negative caching issue, I'd appreciate some comment.